q2_feature_classifier/_vsearch.py

```diff
@@ -19,17 +19,19 @@ def classify_consensus_vsearch(query: DNAFASTAFormat,
                                reference_reads: DNAFASTAFormat,
                                reference_taxonomy: pd.Series,
                                maxaccepts: int = 10,
-                               perc_identity: int = 0.8, strand: str = 'both',
+                               perc_identity: int = 0.8,
+                               query_cov: float = 0.8,
```

Comment:
Looks like you inherited some weirdness here. Should this be `perc_identity: float = 0.8?`

Reply:
thank you for catching that @ChrisKeefe ! Totally bizarre.

Reply:
I just fixed that... let me know if other changes are needed.